CHANGELOG.rst

```diff
@@ -0,0 +1,5 @@
+v0.0.1 (2022-11-11)
```

Comment:
I decided that just manually editing a changelog file is probably all we need instead of any more complex system.

Reply:
That's alright